content/applications/inventory_and_mrp/quality/quality_management/quality_alerts.rst

```diff
@@ -1,81 +1,88 @@
-  the product, and the date assigned
-
-.. image:: quality_alerts/quality-alert-form.png
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+In Odoo *Quality*, *quality alerts* are used to notify quality teams of product defects or other
+issues. Quality alerts can be created from a manufacturing or inventory order, from a work order in
+the *Shop Floor* module, or directly within the *Quality* app.
+
+Create a quality alert
+======================
+
+There are four ways to create a new quality alert:
+
+- Navigate to :menuselection:`Quality -->  Quality Control --> Quality Alerts`, and then click
+  :guilabel:`New` to open a quality alert form.
+- Navigate to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and then
+  select an |MO|. Click the :guilabel:`Quality Alert` button at the top of the |MO| to open a quality
+  alert form in a new page.
+- Open the :menuselection:`Inventory` app, click the :guilabel:`# To Process` button on an inventory
+  order type card (Receipts, Delivery Orders, etc.), and then select an order. Click the
+  :guilabel:`Quality Alert` button at the top of the order to open a quality alert form in a new
+  page.
+- Open the :menuselection:`Shop Floor` module, and then select a work center from the navigation bar
+  at the top of the page. Then, click the :guilabel:`⋮ (three vertical dots)` button at the
+  bottom-right of a work order card to open the :guilabel:`What do you want to do?` menu. Select the
+  :guilabel:`Create a Quality Alert` option from this menu to open a quality alert in a pop-up
+  window.
+
+.. note::
+   Depending on how a new quality alert form is opened, certain fields on the form may already be
+   filled in. For example, if a quality alert is created from a work order card in the *Shop Floor*
+   module, the :guilabel:`Product` and :guilabel:`Work Center` fields will be pre-filled.
+
+After opening a new quality alert form, fill it out as follows:
+
+- Enter a short :guilabel:`Title` that summarizes the issue with the product.
+- If the product issue was discovered at a specific work center, select it from the :guilabel:`Work
+  Center` drop-down menu.
+- If the product is included in a specific picking order, select it from the :guilabel:`Picking`
+  drop-down menu.
+- In the :guilabel:`Team` field, select the quality team that is responsible for managing the
+  quality alert.
+- If a specific employee should be responsible for the quality alert, select them from the
+  :guilabel:`Responsible` drop-down menu.
+- In the :guilabel:`Tags` field, select any tags relevant to the quality alert from the drop-down
+  menu.
+- Use the :guilabel:`Root Cause` field to select the cause of the quality issue, if known.
+- Choose a :guilabel:`Priority` level by selecting a :guilabel:`⭐ (star)` number between one and
+  three. Quality alerts with higher priorities appear at the top of the *Quality Alerts* Kanban board
+  in the *Quality* app.
```

Comment:
I don't believe we capitalize kanban, but I'm not super sure. Also, I thought it might be more helpful to describe where the kanban board is using a menuselection. totally up to u tho!
```suggestion
three. Quality alerts with higher priorities appear at the top of the :guilabel:`Quality Alerts`
  kanban in :menuselection:`Quality --> Quality Control --> Quality Alerts`.
```